app/scripts/app.js

```diff
@@ -506,6 +506,14 @@ angular
       // content (e.g. using :before pseudo-elements).
       $('body').addClass('ios');
     }
+  })
+  .run(function($window, gettextCatalog){
+    gettextCatalog.debug = true;
```

Comment:
we can't have debug mode on by default, this needs to be triggered off something else like a window constant or a local storage value.  To me this is similar to our UI logging, which is turned on by changing local storage values, see: https://github.com/openshift/origin-web-console#enable--disable-console-log-output

Reply:
This is the debug of the angular-gettext itself,which is not very good with openshiftLogLevel. How do you feel about adding global variables to config.js like OPENSHIFT_LANGUAGE?such as   
window.GettextCatalogDebug = true